packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -228,7 +228,9 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
       : undefined;
     const ariaAttrs =
       multiSelect || disabled
-        ? {}
+        ? {
+            role: 'button'
```

Comment:
>role: 'button' [](start = 12, length = 14)

Why is this role button? Also, why does is the role changing depending on if the dropdown is multiselect or disabled?